.github/workflows/ci.yml

```
@@ -20,6 +20,11 @@ jobs:
         uses: baptiste0928/cargo-install@v2.2.0
         with:
           crate: just
+      - name: Install typst-test from github
+        uses: baptiste0928/cargo-install@v2.2.0
+        with:
+          crate: typst-test
+          git: https://github.com/tingerrr/typst-test.git
```

I suggest using the `ci-semi-stable` tag for now.

I'll force push this if I find glaring bugs, but keep it relatively stable to avoid long-running CI from constantly reinstalling new versions which only fix non-critical things.

```suggestion
          git: https://github.com/tingerrr/typst-test.git
          tag: ci-semi-stable
```